sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/training/models/CustomFormModel.java

```diff
@@ -16,6 +16,9 @@
 @Immutable
 public final class CustomFormModel {
 
+    private String displayName;
```

Comment:
Another new constructor is needed to assign value to `displayName` and `modelProperties`